test/reporters/tests/default.test.ts

```diff
@@ -0,0 +1,24 @@
+import { resolve } from 'pathe'
+import { describe, expect, test } from 'vitest'
+import { runVitest } from '../../test-utils'
+
+const root = resolve(__dirname, '../fixtures')
+const run = async (fileFilter: string[]) => await runVitest({ root, reporters: 'default', config: false }, fileFilter)
+
+describe.skipIf(process.env.CI)('default reporter', async () => {
```

Comment:
Do I even need to explain why you shouldn't skip tests in CI?

Reply:
This function only works with TTY, but CI will cause isTTY to be false

Reply:
Aha, yes, I forgot that I could use this very useful method.😅